src/box/lua/net_box.c

```diff
@@ -1151,6 +1151,28 @@ netbox_transport_connect(struct netbox_transport *transport)
 	return -1;
 }
 
+/*
+ * During a worker fiber's yield, the connection could get closed. Testing that
```

Comment:
Nit: the commit msg should probably say `Part of #...`. It's strange, that 2 commits close the same issue. It should also be before `NO_TEST ...` statements

Reply:
Fixed.